src/main/java/net/sf/jabref/logic/l10n/Localization.java

```diff
@@ -15,7 +15,6 @@
 
     public static final String RESOURCE_PREFIX = "l10n/JabRef";
     public static final String MENU_RESOURCE_PREFIX = "l10n/Menu";
-    public static final String BIBTEX = "BibTeX";
```

Comment:
BibTeX is always spelled BibTeX - see https://github.com/koppor/jabref/issues/73 -- BibTeX is a name like "OS/2" or "Linux" or "LaTeX" and must not be translated.

Thus `Localiatzion.lang("BibTex")` is unfortunately wrong in two aspects: 1) casing, 2) requiring translation